adapters/jlog/go.mod

```diff
@@ -1,13 +1,15 @@
 module github.com/luno/workflow/adapters/jlog
 
-go 1.22.3
+go 1.23.4
+
+toolchain go1.23.5
```

Comment:
Think if we're updating, we might want to update the base Go version, and not just the tool one. Think we can stop this from happening by updating the base config to have this in packageRules:
```yaml
 {
      "packageRules": [
        {
          "description": "Opt-out minimum Go version updates: https://github.com/renovatebot/renovate/issues/16715",
          "matchManagers": ["gomod"],
          // We don't want to update the minimum Go version, this isn't _all_ Go updates,
          // it's updates of type "golang" (i.e. Go version).
          "matchDepTypes": ["golang"],
          "enabled": false
        }
      ]
    },
```